lib/themes/dosomething/paraneue_dosomething/templates/campaign/node--campaign.tpl.php

```diff
@@ -433,7 +433,7 @@
             <?php if ($register_voters) : ?>
               <p><?php print $voter_reg_share_modal_text_2 ?></p>
             <?php endif; ?>
-            <?php if ($show_social_share_link) : ?>
+            <?php if ($show_social_share_link == 1) : ?>
```

Comment:
I think the if statement is all you need to express what i thought was a little more readable. how you do the check (i.e checking it as a boolean or if  it is `==1`) is based on what the value of that flag is. So doing `<?php if ($show_social_share_link)` should work.